src/main/java/org/mineacademy/fo/model/SimpleScoreboard.java

```diff
@@ -16,622 +12,556 @@
-
-		return splitText;
-	}
-
-	private static List<String> passMergeThrough(String marge, List<String> text, int maxLength) {
-		final StringBuilder textBuilder = new StringBuilder(marge == null ? "" : marge);
-		final List<Integer> stringLengths = new ArrayList<>();
-		final List<String> newSplitText = new ArrayList<>();
-
-		for (final String line : text) {
-			stringLengths.add(line.length());
-			textBuilder.append(line);
-		}
-
-		String textWithMarge = textBuilder.toString();
-		textWithMarge = textWithMarge.substring(0, Math.min(maxLength, textWithMarge.length()));
-		int previousLength = 0;
-
-		for (int index = 0; index < stringLengths.size(); index++) {
-			int length = stringLengths.get(index);
-
-			if (index == stringLengths.size() - 1)
-				length = Math.max(length, maxLength);
-
-			final String subString = textWithMarge.substring(previousLength, Math.min(textWithMarge.length(), previousLength + length));
-
-			if (!subString.isEmpty())
-				newSplitText.add(subString);
-
-			previousLength += length;
-		}
-
-		return newSplitText;
-	}
+    // ------------------------------------------------------------------------------------------------------------
+    // Fields
+    // ------------------------------------------------------------------------------------------------------------
+
+    private static final String COLOR_CHAR = "\u00A7";
+
+    /**
+     * List of all active scoreboard (added upon creating a new instance)
+     */
+    @Getter
+    private static final List<SimpleScoreboard> registeredBoards = new ArrayList<>();
+
+    /**
+     * Unique chat color identifiers for specific team entries
+     */
+    private static final String[] colors =
+            {"0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "a", "b", "c", "d", "e", "f"};
+
+    /**
+     * Stored scoreboard lines
+     */
+    private final List<String> rows = new ArrayList<>();
+
+    /**
+     * Stored players viewing THIS scoreboard
+     */
+    private final List<UUID> viewers = new ArrayList<>();
+
+    /**
+     * The color theme for key: value pairs such as
+     * Players: 12
+     * Mode: Playing
+     */
+    private final String[] theme = new String[2];
+    /**
+     * The update tick delay
+     */
+    @Getter
+    @Setter
+    private int updateDelayTicks = 20;
+    /**
+     * The title of this scoreboard
+     */
+    @Getter
+    @Setter
+    private String title;
+    /**
+     * The running update task
+     */
+    private BukkitTask updateTask;
+
+    /**
+     * Create a new scoreboard updating every second
+     */
+
+    public SimpleScoreboard() {
+        registeredBoards.add(this);
+    }
+
+    public SimpleScoreboard(final String title) {
+        this(title, 20);
+    }
+
+    /**
+     * Create a new scoreboard with customizable updateDelayTicks
+     */
+    public SimpleScoreboard(final String title, final int updateDelayTicks) {
+        //Scoreboards were introduced in 1.5, objectives were added in 1.8
```

Comment:
Could you put a space after `//`? It just looks better, but that's just my opinion 😅